vtysh/vtysh.c

```diff
@@ -3910,10 +3922,10 @@ void vtysh_init_vty(void)
 	install_element(ISIS_NODE, &vtysh_quit_isisd_cmd);
 	install_element(OPENFABRIC_NODE, &vtysh_exit_fabricd_cmd);
 	install_element(OPENFABRIC_NODE, &vtysh_quit_fabricd_cmd);
-	install_element(KEYCHAIN_NODE, &vtysh_exit_ripd_cmd);
-	install_element(KEYCHAIN_NODE, &vtysh_quit_ripd_cmd);
-	install_element(KEYCHAIN_KEY_NODE, &vtysh_exit_ripd_cmd);
-	install_element(KEYCHAIN_KEY_NODE, &vtysh_quit_ripd_cmd);
```

Comment:
Interesting, why were ripd commands being registered here?

Reply:
my vague recollection( when I investigated this in the past ) was that rip was the only daemon using the keychain commands.  In the future this will change, I imagine, so I believe this is the correct change.

Reply:
This is actually already changed because eigrpd also uses these commands.